src/accelerate/utils/other.py

```diff
@@ -215,9 +215,10 @@ def save(obj, f, save_on_each_node: bool = False, safe_serialization: bool = Fal
 
 # The following are considered "safe" globals to reconstruct various types of objects when using `weights_only=True`
 # These should be added and then removed after loading in the file
+np_core = np._core if Version(np.__version__) >= Version("2.0.0") else np.core
```

Comment:
could you use is_numpy_available function instead ?

Reply:
Done in bd0e022e6b6e0e505cf11b6b8ce2d103f9e96743